lib/api/bucketDeleteTagging.js

```diff
@@ -0,0 +1,50 @@
+const collectCorsHeaders = require('../utilities/collectCorsHeaders');
+const { metadataValidateBucket } = require('../metadata/metadataUtils');
+const { pushMetric } = require('../utapi/utilities');
+const metadata = require('../metadata/wrapper');
+const util = require('node:util');
+
+/**
+ * Bucket Delete Tagging - Delete a bucket's Tagging
+ * @param {AuthInfo} authInfo - Instance of AuthInfo class with requester's info
+ * @param {object} request - http request object
+ * @param {object} log - Werelogs logger
+ * @param {function} callback - callback to server
+ * @return {undefined}
+ */
+async function bucketDeleteTagging(authInfo, request, log, callback) {
+    log.debug('processing request', { method: 'bucketDeleteTagging' });
+
+    let bucket;
+    const bucketName = request.bucketName;
+    const metadataValParams = {
+        authInfo,
+        bucketName,
+        requestType: 'bucketDeleteTagging',
+    };
+
+    try {
+        let promisifyFunc = util.promisify(metadataValidateBucket);
```

Comment:
Same comment from the other PR

Reply:
done https://github.com/scality/cloudserver/pull/4480/files#diff-ecb34075d5365a2b604ea4bcb794846a9c3352a7bdbcd5caebbb8a2c7862466eR19